modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/Xref_update_conf.pm

```diff
@@ -41,22 +41,20 @@ sub default_options {
            ## 'job_factory' parameters
            'species'          => [],
            'antispecies'      => [qw/mus_musculus_129s1svimj mus_musculus_aj mus_musculus_akrj mus_musculus_balbcj mus_musculus_c3hhej mus_musculus_c57bl6nj mus_musculus_cbaj mus_musculus_dba2j mus_musculus_fvbnj mus_musculus_lpj mus_musculus_nodshiltj mus_musculus_nzohlltj mus_musculus_pwkphj mus_musculus_wsbeij drosophila_melanogaster caenorhabditis_elegans saccharomyces_cerevisiae/],
-           'division'         => [],
+           'division'         => 'EnsemblVertebrates',
+           'taxon'            => [],
```

Comment:
This is a bit confusing. There is a 'taxon' parameter in the SpeciesFactory, so the placing of this variable here implies that's what this is for. But this variable isn't passed to that module, it's used to look for an appropriately named mapping file in ensembl/misc-scripts/xref_mapping/XrefMapper, e.g. "wormbase.pm". And as far as I can make out, the code requires a string value, so I don't know why the default value is an empty arrayref.
So naming the parameter 'taxon' doesn't seem right; I would suggest 'mapping_group'.

Reply:
@james-monkeyshines I haven't used the option -taxon. I suggest the pipeline not to have the option at all.

I understand providing the option tried to address was to let us specify -taxon wormbase, have the pipeline recognise the mapper XrefMapper::wormbase. It's not helpful to have this supplied pipeline wide, because we have two custom mappers: "XrefMapper::wormbase" and "XrefMapper::parasite". We mostly have the two parser for the sake of nice gene descriptions, which are different for the two groups.

In a single run I would like to use "XrefMapper::wormbase" for WormBase core species gene descriptions, and "XrefMapper::parasite" for gene descriptions of other species. I don't think we will ever have a run that needs a single mapper, specified through the -taxon option

For now I make a separate mapper for each species by duplicating the mapper I want, and put all of these on PERL5LIB. Thus I can have one pipeline run instead of two, give each species nice descriptions, and I'm happy enough. I don't know what a better solution for this would be - but e.g. if the xref pipeline didn't also create gene descriptions, we wouldn't need two mappers.